src/codec_dav1d.c

```diff
@@ -55,6 +55,11 @@ static avifBool dav1dCodecOpen(avifCodec * codec, avifDecoder * decoder)
         // Give all available threads to decode a single frame as fast as possible
         codec->internal->dav1dSettings.n_frame_threads = 1;
         codec->internal->dav1dSettings.n_tile_threads = AVIF_CLAMP(decoder->maxThreads, 1, DAV1D_MAX_TILE_THREADS);
+#ifdef DAV1D_MAX_POSTFILTER_THREADS
+        // In dav1d 0.8.2 or later (dav1d API version 5.0.1 or later), n_postfilter_threads is
+        // available.
+        codec->internal->dav1dSettings.n_postfilter_threads = AVIF_CLAMP(decoder->maxThreads, 1, DAV1D_MAX_POSTFILTER_THREADS);
+#endif
```

Comment:
We can alternatively check the dav1d version macros, defined in `dav1d.h`:

```
#define DAV1D_API_VERSION_MAJOR 5
#define DAV1D_API_VERSION_MINOR 0
#define DAV1D_API_VERSION_PATCH 1
```

But it will be much more verbose.